docs/guides/intro-to-graphql.md

```diff
@@ -170,15 +170,17 @@ query GetUsers {
   }
 }`;
 
-fetch('/admin/api', {
-  method: 'POST',
+fetch("/admin/api", {
+  method: "POST",
   headers: {
-    'Content-Type': 'application/json',
+    "Content-Type": "application/json",
   },
-  body: {
+  body: JSON.stringify({
```

Comment:
Do we need to stringify a string?

Reply:
We are stringifying an object here: `{query: ...}`.